tests/dataimportcron_test.go

```diff
@@ -95,6 +96,30 @@ var _ = Describe("DataImportCron", Serial, func() {
 				Satisfy(meta.IsNoMatchError),
 			))
 		}
+
+		By("[AfterEach] Delete the DataImportCron under test")
+		// Delete the DataImportCron under test
+		_ = f.CdiClient.CdiV1beta1().DataImportCrons(ns).Delete(context.TODO(), cronName, metav1.DeleteOptions{})
+		Eventually(func() bool {
+			_, err := f.CdiClient.CdiV1beta1().DataImportCrons(ns).Get(context.TODO(), cronName, metav1.GetOptions{})
+			return errors.IsNotFound(err)
+		}, dataImportCronTimeout, pollingInterval).Should(BeTrue(), "DataImportCron was not deleted")
+
+		By("[AfterEach] Wait for all DataImportCrons UpToDate")
+		// Wait for all DataImportCrons to converge
+		dataImportCrons := &cdiv1.DataImportCronList{}
+		err = f.CrClient.List(context.TODO(), dataImportCrons, &client.ListOptions{Namespace: metav1.NamespaceAll})
+		Expect(err).ToNot(HaveOccurred())
+		for _, cronItem := range dataImportCrons.Items {
+			Eventually(func() bool {
+				cron, err = f.CdiClient.CdiV1beta1().DataImportCrons(cronItem.Namespace).Get(context.TODO(), cronItem.Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				condProgressing := controller.FindDataImportCronConditionByType(cron, cdiv1.DataImportCronProgressing)
+				condUpToDate := controller.FindDataImportCronConditionByType(cron, cdiv1.DataImportCronUpToDate)
+				return condProgressing != nil && condProgressing.Status == corev1.ConditionFalse &&
+					condUpToDate != nil && condUpToDate.Status == corev1.ConditionTrue
+			}, dataImportCronConvergeTimeout, pollingInterval).Should(BeTrue(), fmt.Sprintf("Timeout waiting for DataImportCron conditions %q", cronItem.Namespace+"/"+cronItem.Name))
```

Comment:
You can do
```go
.Should(BeTrue(), "Timeout waiting for DataImportCron conditions %q", cronItem.Namespace+"/"+cronItem.Name)
```
Since assertions will call fmt.Sprintf later on your behalf https://github.com/kubevirt/containerized-data-importer/blob/436b808dd4215d644040bb33fdd2b4fd59dd455a/vendor/github.com/onsi/gomega/gomega_dsl.go#L497-L501

Reply:
Thanks! Done